src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleEarningsResilienceCheck.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.LegalEntities;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities
+{
+    public class HandleEarningsResilienceCheck
+    {
+        private readonly IEarningsResilienceCheckService _earningsResilienceCheckService;
+
+        public HandleEarningsResilienceCheck(IEarningsResilienceCheckService earningsResilienceCheckService)
```

Comment:
It might be useful to add a logger so we can be certain the trigger fired, but I can see you log within the service.